src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -50,7 +52,11 @@ private ChatPage openChatForUser(String userName, String password) {
     return new ChatPage().open();
   }
 
-  @Test
+  private void unblockUser(String user){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "unblockUser" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1186)